### PR TITLE
adding Pizza truck simulator as a pygame project

### PR DESCRIPTION
**Pull request by TIDYMOUSE**
- [x] I have updated the documentation accordingly.
- [x] I have added a new Project.
- [x] I have added a README.md file for the Project.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Comment by TIDYMOUSE** [comment]
Hello @sree-hari-s please review my project.  This is one my initial project that I created after learning pygame. Also one of the used song is copy righted, should I change it or is it fine?

**Comment by TIDYMOUSE** [comment]
Also it would be great if you added hacktober fest accepted as labels and assign yourself as the reviewer
@sree-hari-s

**Comment by sree-hari-s** [comment]
> Hello @sree-hari-s please review my project. This is one my initial project that I created after learning pygame. Also one of the used song is copy righted, should I change it or is it fine?

It would be better to change that song.

File: Projects/Pygame-Projects/Pizza Delivery Truck Simulator/README.md
Comment (sree-hari-s) [inline Projects/Pygame-Projects/Pizza Delivery Truck Simulator/README.md:1]: Also just add an information that space button is for jumping. and requirements.txt would also be good addition in the directory.

**Review by sree-hari-s — CHANGES_REQUESTED** [review]

File: Projects/Pygame-Projects/Pizza Delivery Truck Simulator/README.md
Comment (TIDYMOUSE) [reply Projects/Pygame-Projects/Pizza Delivery Truck Simulator/README.md:1]: @sree-hari-s Ive made the required changes 
- [x] added requirements.md
- [x] Used non-copyrighted and royalty free music and images.

If you want some more improvements, please contact me. Thank You.

**Review by sree-hari-s — APPROVED** [review]

**Comment by sree-hari-s** [comment]
Thankyou for your contribution. Please 🌟 the repo and try to contribute more works.

**Pull request merged**